docs/changelog.rst

```diff
@@ -43,6 +43,17 @@ Changelog
   - `Invictus Hyperoin Fund (IHF) <https://www.coingecko.com/en/coins/invictus-hyperion-fund>`__
   - `Flow - Dapper labs (FLOW) <https://www.coingecko.com/en/coins/flow>`__
   - `Lido DAO (LDO) <https://www.coingecko.com/en/coins/lido-dao>`__
+  - `Binance Beacon ETH (BETH) <https://www.cryptocompare.com/coins/beth/overview>`__
+  - `DeXe (DEXE) <https://www.coingecko.com/en/coins/dexe>`__
+  - `Trust Wallet Token (TWT) <https://www.coingecko.com/en/coins/trust-wallet-token>`__
+  - `Meaconcash (MCH) <https://www.coingecko.com/en/coins/meconcash>`__
+  - Singapore Dollar (SGD)
```

Comment:
Remove this from this list and add a new full changelog entry. This is how we have done it in the past for FIAT.